src/main.js

```diff
@@ -0,0 +1,7 @@
+import htmlParser from './htmlParser';
+import postscribe from './postscribe';
+
```

Comment:
Yes, good suggestion (pending of course, what we do related to the comment on splitting out `htmlParser`).